e2e/protractor.conf.js

```diff
@@ -43,6 +43,7 @@ exports.config = {
             },
             args: [
                 '--incognito',
+                '--no-sandbox',
```

Comment:
Why is this needed?

Reply:
I added this flag to be able run it on nodejs pod without it I getting 
`[12:46:41] E/runner - Unable to start a WebDriver session.
[12:46:41] E/launcher - Error: WebDriverError: unknown error: Chrome failed to start: exited abnormally
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
  (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Linux 4.4.111+ x86_64)
    at Object.checkLegacyResponse (/home/jenkins/workspace/iti_activiti-modeling-app_PR-145/node_modules/selenium-webdriver/lib/error.js:546:15)
    at parseHttpResponse (/home/jenkins/workspace/iti_activiti-modeling-app_PR-145/node_modules/selenium-webdriver/lib/http.js:509:13)
    at doSend.then.response (/home/jenkins/workspace/iti_activiti-modeling-app_PR-145/node_modules/selenium-webdriver/lib/http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Status code is 100, no need to save screenshots.
[12:46:41] E/launcher - Process exited with error code 100`

Reply:
possibly this can cause issues on travis